dolphin/drivers/dell_emc/vmax/client.py

```diff
@@ -123,3 +123,58 @@ def list_pools(self):
             LOG.error("Failed to get pool metrics from VMAX: {}".format(err))
             raise exception.StorageBackendException(
                 reason='Failed to get pool metrics from VMAX')
+
+    def list_volumes(self, storage_id):
+
+        try:
+            # List all volumes
+            volumes = self.conn.provisioning.get_volume_list(filters={'data_volume': 'false'})
```

Comment:
make sure, no. of chars in a line not >79

Reply:
Done